model/trace/database.yaml

```diff
@@ -1,10 +1,23 @@
 groups:
-  - id: trace.db.common.query
+  - id: trace.db.common.minimal
     extends: attributes.db.client.minimal
     type: attribute_group
     brief: This group defines the attributes used to perform database client calls.
+    attributes:
+      - ref: db.operation.name
+        sampling_relevant: true
+      - ref: server.address
+        sampling_relevant: true
+      - ref: server.port
+        sampling_relevant: true
+
```

Comment:
mind adding a TODO comment like
```suggestion
# TODO: add db.system once https://github.com/open-telemetry/build-tools/issues/192 is possible
# - ref: db.system
#    sampling_relevant: true  

```
?


Unfortunately we don't mention `db.system` as sampling-relevant in system-specific conventions, but if we add it, it'll appear as 50+ systems in each extension.

Also maybe mention it's being sampling-relevant in the beginning of each doc? E.g.

> db.system MUST be set to "cassandra" and SHOULD be provided at span creation time.